src/index.js

```diff
@@ -36,52 +36,52 @@ class Logentries extends stream.Subscriber {
         [this.token]: this.winston.add(winston.transports.Logentries, { token: this.token })
       };
     } else {
-      console.warn('Logentries logging was not initialized due to a missing token');
+      console.warn('Logentries logging was not initialized due to a missing token'); // eslint-disable-line
     }
   }
 
   /**
-    @function isReady
-    Check if a subscriber is configured properly and is ready to be used
-    @return {boolean}
-  **/
+   * @function isReady
+   * Check if a subscriber is configured properly and is ready to be used
+   * @return {boolean}
+   * */
   isReady() {
     return !!this.winston;
   }
 
   /**
-    @function isEnabled
-    Check if a subscriber will be used
-    Depends on configuration env variables / settings object parameters
-    Checks LOGENTRIES_LOGGING env / settings object param
-    @return {boolean} - if this subscriber is switched on / off
-  **/
+   * @function isEnabled
+   * Check if a subscriber will be used
+   * Depends on configuration env variables / settings object parameters
+   * Checks LOGENTRIES_LOGGING env / settings object param
+   * @return {boolean} - if this subscriber is switched on / off
+   * */
   isEnabled() {
-    const result = ['true', 'false'].includes(process.env.LOGENTRIES_LOGGING) ?
-      process.env.LOGENTRIES_LOGGING === 'true' : this.settings.LOGENTRIES_LOGGING;
+    const result = ['true', 'false'].includes(process.env.LOGENTRIES_LOGGING)
+      ? process.env.LOGENTRIES_LOGGING === 'true' : this.settings.LOGENTRIES_LOGGING;
     return [null, undefined].includes(result) ? true : result;
   }
 
   /**
-    @function handle
-    Process a message and log it if the subscriber is switched on and message's log level is >= than MIN_LOG_LEVEL
-    Finally, pass the message to the next subscriber if any
-    @param message {Object} - message package object
-    @see LoggerStream message package object structure description
-  **/
+   * @function handle
+   * Process a message and log it if the subscriber is switched on and message's log level is >= than MIN_LOG_LEVEL
+   * Finally, pass the message to the next subscriber if any
+   * @param message {Object} - message package object
+   * @see LoggerStream message package object structure description
+   * */
   handle(message) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this function to reduce its Cognitive Complexity from 19 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.dialonce.net/coding_rules#rule_key=javascript%3AS3776)